src/components/LogRow/AnsiiRow/index.tsx

```diff
@@ -13,13 +14,19 @@ const AnsiiRow = forwardRef<any, BaseRowProps>((rowProps, ref) => {
   const { index } = listRowProps;
 
   const line = processedLines[index];
-  // TODO: EVG-17535
-  // This should be replaced with a collapsible component
-  const lineIndex = Array.isArray(line) ? line[0] : line;
-  const lineContent = getLine(lineIndex);
-  if (!lineContent) return null;
+
+  if (Array.isArray(line)) {
+    return (
+      <CollapsedRow ref={ref} {...listRowProps} numCollapsed={line.length} />
```

Comment:
Same goes here
<img width="1236" alt="image" src="https://user-images.githubusercontent.com/4605522/190246968-8b73f9e7-f38b-430b-81ba-3fedf8a7928e.png">

Reply:
Its not an issue when `wrap=true` so that might have something to do with it. 
<img width="804" alt="image" src="https://user-images.githubusercontent.com/4605522/190249801-9ada3abd-118d-4b71-8495-718ee059b568.png">